### ISSUE TITLE
Server crash

### ISSUE THREAD

**Issue by FurryKitten** [Borlea/EchoPet#95]
[crash-2016-07-30_17.42.31-server.txt](https://github.com/Borlea/EchoPet/files/391999/crash-2016-07-30_17.42.31-server.txt)
Using latest 46 build

**Comment by Borlea** [comment]
Thanks for report and giving the full log(plugins), looks like the issue EchoPet had with citizens but obviously that isn't the case so any info you can give related to fake players and whatnot would be great.

**Comment by FurryKitten** [comment]
Yep. I have protocollib and protocolsupport, so players can connect from 1.8 to 1.9. I will check to create a pet from 1.8 and reply there. 

And, you made awesome plugin ^_^
